tensorflow/compiler/tf2tensorrt/convert/ops/tile.cc

```diff
@@ -137,7 +137,8 @@ class ConvertTile : public OpConverterBase<ConvertTile> {
       dynamic_flag = std::any_of(pSize + 1 - dim_adj, pSize + nb_dims,
                                  [](int i) { return i < 0; });
       const int *pMultiplies = replics.weights().GetPointer<int>() + dim_adj;
-      size.d[0] = pMultiplies[0];
+      // for implicit batch mode, this will get overwritten anyway
+      size.d[0] = pMultiplies[0] * pSize[0];
       for (int i = 1 - dim_adj; i < nb_dims; i++)
```

Comment:
Would this be equivalent to:
   not adding line 140-141, but change  the loop to  
   for (int i = 0;  i < nb_dims; i++)
     size.d[i] = pMultiplies[i] * pSize[i];

Reply:
@bixia1  -- dimensions need to be adjusted in the case of implicit batch mode -- there's no getting around that. However, I've refactored the code to make it a bit more readable. Please review!

Reply:
What you have right now aligns with what I suggested: you don't need to pre-assign size.d[0] and then in the loop overwrite the value for implicit batch mode.

Reply:
thanks Bixia!